docs/reference.md

```diff
@@ -430,6 +433,21 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user cannot change this limit when creating the action.
 * A CLI call that exceeds this limit receives an error code corresponding to TOO_MANY_REQUESTS.
 
+### Size of the code per action (Fixed: 48MB)
+* The size limit for the code of an action is 48MB.
+* The limit cannot be changed by the user.
+* An action with too much code will be rejected on trying to create or update it.
+
+### Size of the parameters (Fixed: 1MB)
+* The size limit for the parameters of an action/package/trigger is 1MB.
+* The limit cannot be changed by the user.
+* An entity with too big parameters will be rejected on trying to create or update it.
+
+### Size of the annotations (Fixed: 1MB)
```

Comment:
I don't think we should document annotations, since we don't document what they are and they are for.
I think once we doc annotations, then the limitation can be added to the docs.